src/api/handlers/TutorDeleteHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { APIGatewayEvent, Handler, ProxyResult } from "aws-lambda";
```

Comment:
@bigLucas 
We had a file called _Pet**Delete**Handler_ and a _Tutor**Remove**Handler_, I changed it to keep a pattern.